README.md

```diff
@@ -1,98 +1,518 @@
-6. Delete these instructions and everything up to the _Project Title_ from the README.
-7. Write some great software and tell people about it.
-
-> Keep the README fresh! It's the first thing people see and will make the initial impression.
-
-----
-
-# Project Title
-
-**Description**:  Put a meaningful, short, plain-language description of what
-this project is trying to accomplish and why it matters. 
-Describe the problem(s) this project solves.
-Describe how this software can improve the lives of its audience.
-
-Other things to include:
-
-  - **Technology stack**: Indicate the technological nature of the software, including primary programming language(s) and whether the software is intended as standalone or as a module in a framework or other ecosystem.
-  - **Status**:  Alpha, Beta, 1.1, etc. It's OK to write a sentence, too. The goal is to let interested people know where this project is at. This is also a good place to link to the [CHANGELOG](CHANGELOG.md).
-  - **Links to production or demo instances**
-  - Describe what sets this apart from related-projects. Linking to another doc or page is OK if this can't be expressed in a sentence or two.
-
-
-**Screenshot**: If the software has visual components, place a screenshot after the description; e.g.,
-
-![](https://raw.githubusercontent.com/cfpb/open-source-project-template/master/screenshot.png)
-
-
-## Dependencies
-
-Describe any dependencies that must be installed for this software to work. 
-This includes programming languages, databases or other storage mechanisms, build tools, frameworks, and so forth.
-If specific versions of other software are required, or known not to work, call that out.
+# Pantheon Helpers
+
+## Description
+The CFPB Pantheon of microservices help medium-sized development teams get their work done.
+Pantheon Helpers is a small Node.js library to make it easier to build a microservice in Node.js with CouchDB.
+
+## Features
+This library makes it easy to build an application with the following features:
+
+  * auditing
+  * validation
+  * rapid developement with a build script, and helpers for useful CouchDB and async promise patterns
+  * asyncronous server actions to provide snappy api response times even when the actions kicked off by the api take a long time
+
+Pantheon helpers provide a framework for handling "actions".
+You:
+    1) Define the actions your application will perform.
+    2) Write an action transform function for each action. 
+       The transform function takes an action and a CouchDB document,
+       and idempotently transforms the document as required by the action.
+    3) Write a validation function for each action.
+       The validation function takes the action,
+       the user who performed the action (the actor),
+       and the old and new doc. It throws an error if
+       the action was invalid or unauthorized.
+    4) Write an asyncronous worker handler for any actions that result in side-effects. 
```

Comment:
If it matters to you, this doesn't produce a normal ordered list in the rendered view.